test/src/main/java/org/apache/accumulo/test/RewriteTabletDirectoriesIT.java

```diff
@@ -124,8 +124,10 @@ public void test() throws Exception {
         assertEquals(splits.size() + 1, count);
 
         // This should fail: only one volume
-        assertEquals(1, cluster.exec(RandomizeVolumes.class, "-z", cluster.getZooKeepers(), "-i",
-            getClientInfo().getInstanceName(), "-t", tableName).getProcess().waitFor());
+        assertEquals(1,
+            cluster
+                .exec(RandomizeVolumes.class, "-c", cluster.getClientPropsPath(), "-t", tableName)
+                .getProcess().waitFor());
```

Comment:
Could probably make this format nicer.

Reply:
I tried to fix this but the formatter put it back this way

Reply:
Could force it by disabling the formatter, but I think the formatter is making the correct choice here... it's trying to not split up the second parameter too much. Because the second param is so long, though, it starts on the next line to highlight the fact that it's the start of a new param, rather than part of previous params.